app/assets/stylesheets/application/medications.scss

```diff
@@ -9,6 +9,25 @@
 	height: 150px;
 }
 
+#medication_content {
+  padding: 2.5em 10em;
```

Comment:
Ah ok, let's not use the exact same values as margin/padding then 😆 
I actually like the form in the centre, let's try that out and see what users think.

Thanks for clarifying Rebecca :)